cvmfs/object_fetcher.h

```diff
@@ -195,8 +210,10 @@ class LocalObjectFetcher :
     file_path->clear();
 
     const std::string source = BuildPath(object_hash, hash_suffix);
-    const std::string dest   = temporary_directory_ + "/" +
-                               object_hash.ToString();
+    const std::string dest   = CreateTempPath(temporary_directory_ + "/" +
```

Comment:
Good point. Didn't know that this decompress flavour exists. [Fixed](https://github.com/reneme/cvmfs/commit/50fc4501aabe4830a5c5246c1886fa9edac89508).